stub/src/main/java/io/grpc/stub/CallStreamObserver.java

```diff
@@ -106,9 +106,33 @@
    *   </li>
    * </ul>
    * </p>
+   *
+   * @deprecated Use {@link #disableAutoRequest} instead. This method will be removed.
    */
+  @Deprecated
   public abstract void disableAutoInboundFlowControl();
 
+  /**
+   * Disables automatic flow control where initial tokens are requested when the call is started,
+   * and a token is returned to the peer after a call to the 'inbound' {@link
```

Comment:
although it is previous doc's fault, `token` is unclear. can you add description about token that it is basically number of messages?
one contributor was asking this before in #6561

Reply:
Ya, the problem is that the obvious way to word it is that additional messages are requested from the peer, but that is misleading at best and completely false at worst.  Depending on the size of the message each request may result in the peer being able to send an additional 0 to many messages

If the message read from the buffer is small and the peer is trying to send a large message already, it doesn't actually free the peer to send any messages; if the message read from the buffer is very large, and the peer is waiting to send a bunch of small messages (e.g. 0 byte messages), one request could result in the peer generating and sending thousands of messages.

Without going into the details of how the message based flow control in the API translates to the underlying byte window based HTTP/2 flow control, I'm not really sure how to accurately document this.  For now I've changed it to "message to be read" (without specifying from where).  Thoughts?